quaddtype/numpy_quaddtype/__init__.py

```diff
@@ -43,3 +47,39 @@ def LongDoubleQuadPrecDType():
 bits = get_sleef_constant("bits")
 precision = get_sleef_constant("precision")
 resolution = get_sleef_constant("resolution")
+
+@dataclass
+class QuadPrecFinfo:
+    """Floating-point information for quadruple precision dtype.
+    
+    This class provides information about the floating-point representation
+    used by the QuadPrecDType, similar to numpy.finfo but customized for
+    quad precision arithmetic.
+    """
+    bits: int = int(bits)
+    eps: float = float(epsilon)
```

Comment:
I am really uncomfortable with all the casts to float, since at least for some of them they will undoubtedly over- or underflow. In my mind, the finfo values should have the same dtype as the type we're describing, but I know this would likely be a breaking change; but likely one we need since otherwise e.g. finfo(quad).max don't make any sense